tests/test_runs/test_run_functions.py

```diff
@@ -354,6 +356,21 @@ def test_run_and_upload_pipeline_dummy_pipeline(self):
                                     ('dummy', DummyClassifier(strategy='prior'))])
         self._run_and_upload(pipeline1, '62501')
 
+    @pytest.mark.skipif(LooseVersion(sklearn.__version__) < "0.20",
```

Comment:
Is there a reason to use a method from pytest here instead of the respective one from unittest?

Reply:
It was my understanding that unittest is no longer supported and that we move to pytest? Correct me if I am wrong

Reply:
No, it's actually `nosetests`, `unittest` is a standard module (with somewhat limited functionality, that's we used nose and now use pytest).

Reply:
Cool, I replaced them.